gluster-ansible.spec

```diff
@@ -1,10 +1,10 @@
 %global rolesdir %{_sysconfdir}/ansible/roles/gluster.ansible
 %global docdir %{_datadir}/doc/gluster.ansible
-%global buildnum 2
+%global buildnum 5
 
-Name:      gluster-ansible-roles
-Version:   1.0.0
-Release:   2%{?dist}
+Name:      gluster-ansible
```

Comment:
why is this part of pre-tasks change?

Reply:
This was a earlier commit which was done to do a build. And when the PR is created it included earlier commits as well.